### PR TITLE
[PLUGIN-670] Fixed partition boundaries query. Allow different projects to be set ...

### PR DESCRIPTION
**Pull request by sechegaray**
…for temporary tables

### CODE REVIEW

File: src/main/java/io/cdap/plugin/gcp/bigquery/source/PartitionedBigQueryInputFormat.java [hunk 1]
```diff
@@ -115,12 +115,10 @@ private void processQuery(JobContext context) throws IOException, InterruptedExc
       TableReference exportTableReference = createExportTableReference(type, datasetProjectId, datasetId,
                                                                        temporaryTableName, configuration);
       runQuery(configuration, bigQueryHelper, projectId, exportTableReference, query, location);
-      if (type == Type.VIEW || type == Type.MATERIALIZED_VIEW) {
-        configuration.set(BigQueryConfiguration.INPUT_PROJECT_ID_KEY,
-                          configuration.get(BigQueryConstants.CONFIG_VIEW_MATERIALIZATION_PROJECT));
-        configuration.set(BigQueryConfiguration.INPUT_DATASET_ID_KEY,
-                          configuration.get(BigQueryConstants.CONFIG_VIEW_MATERIALIZATION_DATASET));
-      }
+      configuration.set(BigQueryConfiguration.INPUT_PROJECT_ID_KEY,
```
Comment (fernst) [inline src/main/java/io/cdap/plugin/gcp/bigquery/source/PartitionedBigQueryInputFormat.java:120]: nit: Add a comment saying that these values come from the default sink config and can be overridden via config

File: src/main/java/io/cdap/plugin/gcp/bigquery/source/PartitionedBigQueryInputFormat.java [hunk 2]
```diff
@@ -291,20 +284,22 @@ private String generateTimePartitionCondition(StandardTableDefinition tableDefin
     }
 
     if (partitionFromDate != null) {
+      String columnFromTimestampName = columnName;
       if (LegacySQLTypeName.DATE.equals(columnType)) {
-        columnName = "TIMESTAMP(\"" + columnName + "\")";
+        columnFromTimestampName = "TIMESTAMP(`" + columnFromTimestampName + "`)";
```
Comment (fernst) [inline src/main/java/io/cdap/plugin/gcp/bigquery/source/PartitionedBigQueryInputFormat.java]: `columnFromTimestampName` is a confusing name. Can you think of a better/simpler to understand column name?

**Review by fernst — COMMENTED** [review]
Asked a question. Code looks good but I wonder if we can add some comments and a better variable name

File: src/main/java/io/cdap/plugin/gcp/bigquery/source/PartitionedBigQueryInputFormat.java (hunk 1 above)
Comment (sechegaray) [reply src/main/java/io/cdap/plugin/gcp/bigquery/source/PartitionedBigQueryInputFormat.java:120]: done

**Review by fernst — APPROVED** [review]
LGTM

**Pull request merged**